var/spack/repos/builtin/packages/py-onnx-runtime/package.py

```diff
@@ -126,4 +132,11 @@ def cmake_args(self):
 
     @run_after("install")
     def install_python(self):
-        PythonPackage.install(self, self.spec, self.prefix)
+        """Install everything from build directory."""
+        # Truncated version of PythonPipBuilder.install
+
+        args = PythonPipBuilder.std_args(self) + ["--prefix=" + prefix, "."]
+
+        pip = inspect.getmodule(self).pip
+        with fs.working_dir(self.build_directory):
+            pip(*args)
```

Comment:
```suggestion
        args = std_pip_args + ["--prefix=" + prefix, "."]
        with working_dir(self.build_directory):
            pip(*args)
```
This should be sufficient. This is also what we do in `py-tensorflow` and many other packages like this.